thunder/benchmarks/benchmark_litgpt.py

```diff
@@ -307,13 +285,47 @@ def setup_distributed(self, model):
                     sharding_strategy=sharding_strategy,
                     bucketing_strategy=bucketing_strategy,
                 )
+            else:
+                if self.distributed_mode == "fsdp2":
+                    raise ValueError(f"`fsdp2` is not available for `thunder.jit`.")
         else:
             if self.distributed_mode == "ddp":
                 model = torch.nn.parallel.DistributedDataParallel(
                     model,
                     device_ids=[local_rank],
                     bucket_cap_mb=self.ddp_bucket_size,
                 )
+            elif self.distributed_mode == "fsdp2":
```

Comment:
The bucketing option is unused in this path. Should there be an error or warning?

Reply:
going with warning